src/Conversion/ONNXToKrnl/Math/Softmax.cpp

```diff
@@ -166,6 +168,10 @@ void emitInstForSoftmax<ONNXSoftmaxV11Op>(ConversionPatternRewriter &rewriter,
     SmallVector<IndexExpr, 4> outerUbs;
     for (int i = 0; i < axis; ++i)
       outerUbs.emplace_back(create.krnlIE.getShapeAsDim(input, i));
+    if (enableParallel) {
```

Comment:
For softmax, don't we have to check that the outer loop is not one on which the softmax operation is performed?

```
Attributes
axis - INT (default is '-1'): Describes the dimension Softmax will be performed on. Negative value means counting dimensions from the back. Accepted range is [-r, r-1] where r = rank(input).
```

Please double check.